src/Components/CreateImageWizard/steps/Packages/Packages.tsx

```diff
@@ -1087,9 +1078,7 @@ const Packages = () => {
     return (
       (pkg.type === 'module' &&
         modules.some((module) => module.name === pkg.module_name) &&
-        !modules.some((p) => p.stream === pkg.stream)) ||
-      false
-    );
+        !modules.some((p) => p.stream === pkg.stream)));
```

Comment:
I think that `|| false` is not needed here. :thinking: However, I would hate breaking something before summit:D